src/Parsing/ParserState.php

```diff
@@ -251,35 +251,35 @@ public function peek($length = 1, $offset = 0): string
     }
 
     /**
-     * @param int $mValue
```

Comment:
Looks like the type here should be `int|string`, but that's beyond the scope of this PR.